.licensei.toml

```diff
@@ -20,6 +20,7 @@ ignored = [
     # Vanity URLS :\
     "cloud.google.com/go",
     "emperror.dev/errors",
+    "gopkg.in/square/go-jose.v2",
```

Comment:
**Note**: I think this should be part of a different PR.

Reply:
I haven't changed anything regarding JWT libs in this PR, but the build is red without this line, probably some indirect dependency changed in the background: https://app.circleci.com/pipelines/github/banzaicloud/banzai-cli/1370/workflows/fba26aae-bfba-4974-86b5-301da0a90381/jobs/2478

Reply:
Interesting, because the corresponding dependency was added more than a year ago. Has the packaged changed its license?

I'm fine merging it this way then (though let's make sure we don't overlook a possibly problematic license in case it did change).

(A separate fix PR and this user_sess removal PR would be preferred to avoid possibly reverting the license check fix in case we need to revert the user_sess removal code, but I don't want to split hairs over this, just keep in mind for the future.)